admin/spec/controllers/questions.spec.js

```diff
@@ -274,5 +283,21 @@ describe('Questions controller', () => {
       expect(pupilLoginEventService.storeLogonEvent).toHaveBeenCalled()
       done()
     })
+    it('returns error if the checkWindow service throws', async (done) => {
+      const req = goodReq
+      const controller = setupController({
+        'check-window.service.hasActiveCheckWindow': function () { return Promise.reject(new Error('a mock')) }
+      })
+      try {
+        await controller.getQuestions(req, res)
+      } catch (error) {
+        expect('questions controller not to').toBe('error')
+        done()
```

Comment:
Do you need this `done()`.  Won't it make the callback but carry on execution anyway?